src/test/java/spoon/support/reflect/CtModifierHandlerTest.java

```diff
@@ -0,0 +1,67 @@
+package spoon.support.reflect;
+
+import org.junit.Before;
+import org.junit.Test;
+import spoon.Launcher;
+import spoon.reflect.declaration.ModifierKind;
+import spoon.reflect.factory.Factory;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
+
+public class CtModifierHandlerTest {
+
+    LinkedHashSet<ModifierKind> inputModifierKinds = new LinkedHashSet<>(Arrays.asList(
+            ModifierKind.STATIC,
+            ModifierKind.PUBLIC,
+            ModifierKind.ABSTRACT,
+            ModifierKind.NATIVE
+    ));
+    LinkedHashSet<CtExtendedModifier> inputExtendedModifiers = inputModifierKinds.stream().map(CtExtendedModifier::new).collect(Collectors.toCollection(LinkedHashSet::new));
```

Comment:
Maybe chop this down?
```suggestion
    LinkedHashSet<CtExtendedModifier> inputExtendedModifiers = inputModifierKinds
        .stream()
        .map(CtExtendedModifier::new)
        .collect(Collectors.toCollection(LinkedHashSet::new));
```

Reply:
I'll make those changes separately, since there's other stuff I'm changing as well